### PR TITLE
Fact categories backend.

### PR DESCRIPTION
**Pull request by mmcdermott**
This should be smoke tested with a client before deployed.

### CODE REVIEW

File: app/controllers/fact_categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+class FactCategoriesController < ApplicationController
+  before_action :authenticate, :set_variables, :check_authorization
+
+  # TODO(Ozzie): Should the api be .../fact_categories or .../categories ?
```
Comment (OAGr) [inline app/controllers/fact_categories_controller.rb]: I like '/fact_catogories'

Comment (mmcdermott) [inline app/controllers/fact_categories_controller.rb]: K, cool. That's what it is, so I'll remove this TODO in the next push.

File: app/controllers/fact_categories_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  # POST /organizations/:organization_id/fact_categories.json
+  def create
+    if @fact_category.save
+      render json: FactCategoryRepresenter.new(@fact_category).to_json
+    else
+      render json: @fact_category.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /organizations/:organization_id/fact_categories/:id
```
Comment (OAGr) [inline app/controllers/fact_categories_controller.rb]: why both lines?  This won't be json ever, i assume.

Comment (mmcdermott) [inline app/controllers/fact_categories_controller.rb]: I was just mimicing what we do in other controllers, but I can remove the .json line.

**Pull request merged**